src/ess/reduce/nexus/types.py

```diff
@@ -95,9 +91,9 @@ class TransmissionRun(Generic[ScatteringRunType]):
 """Identifier for an arbitrary monitor"""
 Monitor5 = NewType('Monitor5', int)
 """Identifier for an arbitrary monitor"""
-Incident = NewType('Incident', int)
+IncidentMonitor = NewType('IncidentMonitor', int)
```

Comment:
Added the monitor suffix here since these now show up as `MyComponent[IndicidentMonitor]` and it may be confusing without.